### ISSUE TITLE
Administrator SQL result view doesn't sanitize data

### ISSUE THREAD

**Issue by linkdata** [Starcounter/Starcounter#3589]
Got this when selecting randomly generated data.
Aren't we sanitizing and escaping data before trying to render it?
If not, injecting scripts into the administrator UI is trivial.

```
SyntaxError: Unexpected token , in JSON at position 12427 at Object.parse (native) at qc
(http://127.0.0.1:8181/app/vendor/angular/angular.min.js:14:245) at Zb 
(http://127.0.0.1:8181/app/vendor/angular/angular.min.js:76:423) at 
http://127.0.0.1:8181/app/vendor/angular/angular.min.js:77:283 at r 
(http://127.0.0.1:8181/app/vendor/angular/angular.min.js:7:302) at Zc 
(http://127.0.0.1:8181/app/vendor/angular/angular.min.js:77:265) at c 
(http://127.0.0.1:8181/app/vendor/angular/angular.min.js:78:414) at 
http://127.0.0.1:8181/app/vendor/angular/angular.min.js:112:113 at n.$eval 
(http://127.0.0.1:8181/app/vendor/angular/angular.min.js:126:15) at n.$digest 
(http://127.0.0.1:8181/app/vendor/angular/angular.min.js:123:106)
```

**Comment by dan31** [comment]
cc @miyconst @andwah (expecting @miyconst get the point on the issue)

**Comment by miyconst** [comment]
If I understood the issue correct, then it's definitely a bug. All the data displayed has to be escaped to avoid any HTML or JavaScript injections. And security is not even a main point in here. Injected scripts and/or HTML elements might break entire administrator page.

The stack trace provided by @linkdata makes me think that this is something wrong with administrator script, not injected data being executed.

@andwah should know more.

**Comment by andwah** [comment]
I will take a look.

**Comment by linkdata** [comment]
This may be another manifestation of https://github.com/Starcounter/Starcounter/issues/3585

**Comment by chrhol** [comment]
Solved by the same fix as in #3585.
